resteasy-client/src/test/java/org/jboss/resteasy/client/jaxrs/engines/ManualClosingApacheHttpClient43EngineTest.java

```diff
@@ -0,0 +1,47 @@
+package org.jboss.resteasy.client.jaxrs.engines;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ManualClosingApacheHttpClient43EngineTest {
+
+   @Before
+   public void unsetMemThresholdProperty() {
+      System.clearProperty(ManualClosingApacheHttpClient43Engine.FILE_UPLOAD_IN_MEMORY_THRESHOLD_PROPERTY);
+   }
```

Comment:
Ideally, the test should leave the environment in the same state it was before the run. So we should probably store the value of the property before the execution and restore it afterwards.